sql/create_links.sql

```diff
@@ -4,9 +4,13 @@
 
 CREATE TABLE public.links
 (
-    address VARCHAR(46) NOT NULL, --ethereum address
+    address VARCHAR(46) NOT NULL, -- ethereum address
     did VARCHAR(64) NOT NULL, -- did,
     consent text,
+    type VARCHAR(64), -- account type(ethereum-eoa, erc1271)
+    chainId VARCHAR(30),
+    contractAddress VARCHAR(46),
+    timestamp INT(11) unsigned NOT NULL DEFAULT 0,
```

Comment:
actually posgresql db right now, think this has to change to 
```timestamp INT NOT NULL DEFAULT 0```

Reply:
fixed

Reply:
Thanks @jsidorenko. Will let zach have a final look, will try to have this reviewed and released today or tomorrow!